packages/rum-core/src/domain/rumEventsCollection/action/getActionNameFromElement.ts

```diff
@@ -1,36 +1,38 @@
 import { safeTruncate } from '@datadog/browser-core'
 
-export function getActionNameFromElement(element: Element): string {
+/**
+ * Get the action name from the attribute 'data-dd-action-name' on the element or any of its parent.
+ * This can be customized by the setting the actionNameAttribute configuration option.
+ */
+const PROGRAMMATIC_ATTRIBUTE = 'data-dd-action-name'
+
+export function getActionNameFromElement(element: Element, actionNameAttribute = PROGRAMMATIC_ATTRIBUTE): string {
   // Proceed to get the action name in two steps:
   // * first, get the name programmatically, explicitly defined by the user.
   // * then, use strategies that are known to return good results. Those strategies will be used on
   //   the element and a few parents, but it's likely that they won't succeed at all.
   // * if no name is found this way, use strategies returning less accurate names as a fallback.
   //   Those are much likely to succeed.
   return (
-    getActionNameFromElementProgrammatically(element) ||
+    getActionNameFromElementProgrammatically(element, actionNameAttribute) ||
```

Comment:
What about always checking `data-dd-action-name`, and checking *extra* attributes based on the configuration? This would allow library authors to use the constant attribute name without worrying that it could be changed by the application.

(Illustration: I'm a frontend dev, and I want to share my uikit library to the world. I want to support Datadog Browser SDK and provide action names on buttons out of the box. I want to use a constant attribute for it, and avoid asking the user for its "actionNameAttribute" value.)